sync.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# TODO create sub-directories per opendaylight branch
+
+if [ -z "$1" ]; then
+  echo "USAGE: sync.sh <PATH-TO-YOUR-opendaylight.git/releng/autorelease>"
+  exit
+fi
+set -ex
+
+pushd .
+cd $1
+git pull
+git submodule init
+git submodule update
+git submodule foreach git reset --hard
+git submodule foreach git checkout origin/master
+git submodule foreach git pull --ff-only origin master
+popd
+
+git rm experimental/odp/*
```

Comment:
It seems a shame to lose the history of models as they evolve, as I think that is what will happen with this approach. I realize this is the easiest way to do it, but would you consider something that maintains version history for files that survive from one upload to the next? This could, for example, assist developers in adopting changes.

Reply:
Further, do developers need to access release-specific instances of models from here? This is also made harder unless you have some form of tagging strategy associated.

Reply:
> > It seems a shame to lose the history of models as they evolve
> 
> @einarnn the history of models as they evolve is actually preserved like this, see e.g. [this change over in the #558 example](https://github.com/YangModels/yang/pull/558/files#diff-c94b4b3f66624c4b660d6be18b94e619)

I suspect @einarnn would like to have each YANG model’s history preserved, _i.e._ the commit logs from each `.yang` file, as you would get by using `filter-branch` and a merge with unrelated histories.

Reply:
@skitt thanks, I see what you mean, I think. @einarnn can we clarify what you / this project are after? My current script does preserve the history in this repo, [see this example](https://github.com/YangModels/yang/commits/c5194855843e5f6df3b311dec13c7ec650955819/experimental/odp/flow-topology-discovery.yang). It does not "preserve" the history from the "source" ODL Git repo in this repo, so e.g. on that `flow-topology-discovery.yang` example you don't get the `git log` from the ODL Git repo for that file... I wrote _preserve_ in quote because that more of a ... "transposition" than a "preservation" - from one git repo to another. Doing that is beyond what I am both capable, interested in and have time for here - sorry! If that is the requirement, then what I am suggesting here is the wrong approach; should that be a hard requirement (and doing this as a first step is not of interest to this community), then I would abandon this PR and let someome else step up to contribute a better alternative instead.